napari_allencell_annotator/util/image_utils.py

```diff
@@ -34,8 +35,8 @@ def __init__(self, filepath: Path):
         else:
             self._image = BioImage(filepath, reader=bioio_imageio.Reader)
 
-    def get_image_data(self) -> np.ndarray:
+    def get_image_data(self) -> da.Array:
```

Comment:
i think `get_dask_data` makes more sense

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/77/commits/04d7002f0f1b26a569a1ff58a94d6aad63efcc65